test/e2e/run_tests.sh

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
 
 TESTS_DIR="$1"
+TESTS_RUN_LONG="${long_tests:-no}"
```

Comment:
Just checking: the intention of long_tests is for it to be an environment variable or specified on the command line before the command, right?

Reply:
In the perspective of the script, there is no difference between an exported environment variable and a variable that is specified on the command line before the command. Both will work. (In case of name collision, the definition on the command line takes precedence.)